gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Apr 10 15:27:10 PDT 2013
+#Mon Apr 25 17:06:04 EEST 2016
```

Comment:
Can we leave this file unchanged? Upgrading the gradle version shouldn't be required.